### PR TITLE
Add eseal theme

### PR DESCRIPTION
**Pull request by isinyaaa**

### CODE REVIEW

**Review by derekstavis — APPROVED** [review]

File: packages/eseal [hunk 1]
```diff
@@ -0,0 +1,4 @@
+type = plugin
```
Comment (derekstavis) [inline packages/eseal:1]: This should be `theme` instead

**Review by derekstavis — CHANGES_REQUESTED** [review]
Approved mistakenly first. Please make the requested change before we can merge

File: packages/eseal (hunk 1 above)
Comment (scorphus) [inline packages/eseal:1]:
```suggestion
type = theme
```

**Review by scorphus — CHANGES_REQUESTED** [review]

**Review by scorphus — APPROVED** [review]
@isinyaaa it would be helpful adding a screenshot of eseal 🙂

**Comment by isinyaaa** [comment]
> @isinyaaa it would be helpful adding a screenshot of eseal 🙂

okay, will do :)

**Comment by isinyaaa** [comment]
@scorphus you mean adding it here? I don't see any way to add it to the repo.

anyway, there you go:

image

**Review by derekstavis — APPROVED** [review]

**Pull request merged**